modules/ols-creating-lightspeed-custom-resource-file-using-web-console.adoc

```diff
@@ -34,17 +34,40 @@ spec:
         type: openai
         credentialsSecretRef:
           name: credentials
-        url: "https://api.openai.com/v1"
+        url: https://api.openai.com/v1
         models:
           - name: gpt-3.5-turbo
   ols:
     defaultModel: gpt-3.5-turbo
     defaultProvider: myOpenai
 ----
 +
-.Example {azure-openai} custom resource file
+.Example {rhoai} CR file
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: ols.openshift.io/v1alpha1
+kind: OLSConfig
+metadata:
+  name: cluster
+  namespace: openshift-lightspeed
+spec:
+  llm:
+    providers:
+    - credentialsSecretRef:
+        name: openai-api-keys
+      models:
+      - name: granite-8b-code-instruct-128k
+      name: red_hat_openshift_ai
+      type: red_hat_openshift_ai_vllm
+      url: <https://granite-8b-code-instruct.my-domain.com:443/v1> <1>
```

Comment:
Confused by the angle brackets. Is this meant to be an example that they swap out or a verbatim URL they must use?

If verbatim:
```suggestion
      url: https://granite-8b-code-instruct.my-domain.com:443/v1 <1>
```

If example, maybe something generic like:
```suggestion
      url: <url> <1>
```
and then down in the callout provide the fuller example:
```
<1> The URL endpoint must end with `v1` to be valid. For example, `\https://granite-8b-code-instruct.my-domain.com:443`.
```
Note the `\` before the `https://` there, so that it doesn't render as a link (per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#example-urls)).

Or if only `my-domain` is the part that is variable:
```suggestion
      url: https://granite-8b-code-instruct.<your_domain>.com:443/v1 <1>
```

Reply:
It was the second use case you outlined. Thanks!